src/datta_lab_to_nwb/markowitz_gillis_nature_2023/preconversion/extract_metadata.py

```diff
@@ -292,9 +323,11 @@ def get_session_name(session_df):
     session_name : str
         Session name.
     """
-    session_names = set(session_df.session_name[session_df.session_name.notnull()]) | set(
-        session_df.SessionName[session_df.SessionName.notnull()]
-    )
+    session_names = set(session_df.SessionName[session_df.SessionName.notnull()])
+    try:
+        session_names = session_names.union(set(session_df.session_name[session_df.session_name.notnull()]))
+    except AttributeError:  # No session name column
```

Comment:
Is it possible to do something like `session_name in session_df`?

We generally try to avoid `try/except` clauses wherever possible